ashlar/migrations/0014_auto_20150929_1551.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.gis.db.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ashlar', '0013_auto_20150514_1712'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='record',
+            options={'ordering': ('-created',)},
+        ),
+        migrations.AlterField(
```

Comment:
Hmm, if you check out `develop` and generate migrations, does it still generate this? It might have been a change from a while ago that didn't get a migration when it should have. If so, it's probably safe to include it but I'd split it out into a separate commit.